### PR TITLE
asdfasdf

### PR DESCRIPTION
**Pull request by dvarchev**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/dvarchev/framework/pull/1" target="_blank">ReviewNinja

File: asdfsadf [hunk 1]
```diff
@@ -0,0 +1 @@
+asdfsadfasdf
```
Comment (dvarchev) [inline asdfsadf:1]: Why, oh, why!!!

Comment (dvarchev) [inline asdfsadf:1]: !fix

Comment (dvarchev) [inline asdfsadf:1]: !fixed

**Comment by dvarchev** [comment]
!ninjastar

**Comment by dvarchev** [comment]
:+1:

**Comment by dvarchev** [comment]
!star

**Pull request merged**